src/containers/cookie-consent-banner/CookieConsentBanner.tsx

```diff
@@ -0,0 +1,70 @@
+import DoneIcon from '@mui/icons-material/Done'
+import { Box, Typography } from '@mui/material'
+import Link from '@mui/material/Link'
+import { FC, useState } from 'react'
+import { createPortal } from 'react-dom'
+import { Trans, useTranslation } from 'react-i18next'
+import { useLocation } from 'react-router-dom'
+import AppButton from '~/components/app-button/AppButton'
+import HashLink from '~/components/hash-link/HashLink'
+import { styles } from '~/containers/cookie-consent-banner/CookieConsentBanner.styles'
+import { guestRoutes } from '~/router/constants/guestRoutes'
+import {
+  getFromLocalStorage,
+  setToLocalStorage
+} from '~/services/local-storage-service'
+
+const CookieConsentBanner: FC = () => {
+  const { t } = useTranslation()
+  const { pathname } = useLocation()
+  const [cookieConsent, setCookieConsent] = useState<boolean>(() => {
+    return getFromLocalStorage('cookieConsent') || false
+  })
+
+  const handleButtonClick = () => {
+    setCookieConsent(true)
+    setToLocalStorage('cookieConsent', true)
+  }
+
+  if (cookieConsent || pathname.slice(1) === guestRoutes.privacyPolicy.path) {
+    return null
+  }
+
+  return createPortal(
+    <Box sx={styles.root}>
+      <Typography sx={styles.notice}>
+        <Trans
+          components={[
+            <Link
+              component={HashLink}
+              key={guestRoutes.privacyPolicy.path}
+              sx={styles.link}
+              target='_blank'
+              to={guestRoutes.privacyPolicy.path}
+            />,
+            <Link
+              component={HashLink}
+              key={'2'}
```

Comment:
Can we make more semantic keys?

Reply:
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/32570823/d4456588-858e-45d2-8996-ca1dbb7571d0)

that's because these two links lead to the same page now, we have only the cookie policy page implemented.